cmd/describeaccount_test.go

```diff
@@ -217,3 +217,22 @@ func TestDescribeAccount_JSTiers(t *testing.T) {
 	require.Contains(t, out, " | optional")
 	require.Contains(t, out, " | 99")
 }
+
+func TestDescribeAccount_Callout(t *testing.T) {
+	ts := NewTestStore(t, "test")
+	defer ts.Done(t)
+
+	ts.AddAccount(t, "A")
+
+	_, uPK, _ := CreateUserKey(t)
+	_, aPK, _ := CreateAccountKey(t)
+	_, _, err := ExecuteCmd(createEditAuthorizationCallout(),
+		"--auth-user", uPK,
+		"--allowed-account", aPK)
```

Comment:
Can this be an array also? Or comma separated I mean?

Reply:
both, and you can supply seeds or paths to files